mongodbatlas/fw_resource_mongodbatlas_alert_configuration_test.go

```diff
@@ -312,6 +313,33 @@ func TestAccConfigRSAlertConfiguration_importBasic(t *testing.T) {
 	})
 }
 
+func TestAccConfigRSAlertConfiguration_importIncorrectId(t *testing.T) {
+	var (
+		orgID        = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName  = acctest.RandomWithPrefix("test-acc")
+		resourceName = "mongodbatlas_alert_configuration.test"
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheckBasic(t) },
+		ProtoV6ProviderFactories: testAccProviderV6Factories,
+		CheckDestroy:             testAccCheckMongoDBAtlasAlertConfigurationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMongoDBAtlasAlertConfigurationConfig(orgID, projectName, true),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportStateIdFunc:       testAccCheckMongoDBAtlasAlertConfigurationImportStateIncorrectIDFunc(resourceName),
```

Comment:
could we use ImportStateId directly here? (defining a fixed value with the incorrect format)

Reply:
thx, changed in a new commit